alpine-nix-rails-nginx/etc/nginx/nginx.conf

```diff
@@ -79,20 +84,24 @@ http {
 
   # Catch-all domain
   server {
-    listen      80 default_server;
-    listen [::]:80 default_server;
+    listen      8080 default_server;
+    listen [::]:8080 default_server;
+
+    access_log /var/log/nginx/access.log;
+    error_log  /var/log/nginx/error.log;
```

Comment:
It turns out that it does inherit from the `main` context, which is nice. I also found out that `access_log` can be specified once in the `http` context, so I moved it there too.